tools/install_intel_oneAPI_windows.bat

```diff
@@ -0,0 +1,21 @@
+REM Reference - https://github.com/oneapi-src/oneapi-ci/blob/master/scripts/install_windows.bat
+REM SPDX-FileCopyrightText: 2022 Intel Corporation
+REM
+REM SPDX-License-Identifier: MIT
+
+
+set URL=%1
+set COMPONENTS=%2
+
+
+curl.exe --output %TEMP%\webimage.exe --url %URL% --retry 5 --retry-delay 5
```

Comment:
Please comment what this does, e.g.
```suggestion
:: download installer from intel
curl.exe --output %TEMP%\webimage.exe --url %URL% --retry 5 --retry-delay 5
```

Reply:
This whole file is taken from https://github.com/oneapi-src/oneapi-ci. I have added link to the original file in the first line. See https://github.com/scipy/scipy/pull/20985/commits/e979da6bd1d0e629f923f3edc6961a4f9447063b